doc/source/jupyterhub/customizing/user-environment.md

```diff
@@ -69,28 +69,157 @@ If you'd like users to select an environment from **multiple docker images**,
 see {ref}`multiple-profiles`.
 ```
 
+(user-interfaces)=
+
+## Selecting a user interface
+
+[JupyterLab][] is the new user interface for Jupyter,
+which is meant to replace the classic notebook user interface (UI).
+While users already can interchange `/tree` and `/lab` in the URL to switch between
+the classic UI and JupyterLab if both are installed,
+Deployments using JupyterHub 1.x and earlier default to the classic UI,
+while 2.0 will make JupyterLab the default.
```

Comment:
```suggestion
Users already can interchange `/tree` and `/lab` in the URL to switch between
the classic UI and JupyterLab if both are installed.
Deployments using JupyterHub 1.x and earlier default to the classic UI,
while JupyterHub 2.0 makes JupyterLab the default.
```

Reply:
Are both these variations reasonable English? The latter feels more right to me.

`Users already can interchange` or `Users can already interchange`

Reply:
The latter sounds better, I'll update my suggestion.